### ISSUE TITLE
country-budget-items allows a mix of text and iati sub elements

### ISSUE THREAD

**Issue by Bjwebb** [IATI/IATI-Schemas#27]
In the new schema, XML like this is valid:

``` xml
<country-budget-items vocabulary="1">
   IATI
  <budget-item code="1.3.3" percentage="100">
    Accountability - planning
    <description>Accountability - planning</description>
  </budget-item>
</country-budget-items>
```

This is because country-budget-items has `mixed="true"`. This is different to most other elements that have subelements in the schema. Is this a bug?

**Comment by Bjwebb** [comment]
We think that this is a bug, but fixing it will break backwards compatibility. We intend to fix this in 2.01. I'll post on the support forum shortly.

**Comment by Bjwebb** [comment]
http://support.iatistandard.org/entries/47428736-Don-t-allow-mixed-XML-and-text-in-country-budget-items

**Comment by Bjwebb** [comment]
We're currently planning to change the schema such that no elements are mixed -https://github.com/IATI/IATI-Schemas/issues/112 - which would also fix this issue.

**Comment by Bjwebb** [comment]
Due to the changes in https://github.com/IATI/IATI-Schemas/issues/112 this issue has now been fixed in 2.01.
